config/core/build-configs.yaml

```diff
@@ -506,134 +506,12 @@ fragments:
-      - 'CONFIG_MFD_INTEL_LPSS_ACPI=m'
+      - 'CONFIG_IGB=y'
       - 'CONFIG_MFD_INTEL_LPSS_PCI=y'
-      - 'CONFIG_MMC_SDHCI_ACPI=y'
       - 'CONFIG_MMC_SDHCI_PCI=y'
       - 'CONFIG_MMC_SDHCI=y'
       - 'CONFIG_MMC=y'
-      - 'CONFIG_NFS_FS=y'
-      - 'CONFIG_NFS_V2=y'
-      - 'CONFIG_NFS_V3_ACL=y'
-      - 'CONFIG_NFS_V3=y'
-      - 'CONFIG_NFS_V4=y'
-      - 'CONFIG_NVME_CORE=y'
-      - 'CONFIG_PINCTRL_AMD=y'
-      - 'CONFIG_PINCTRL_BAYTRAIL=y'
-      - 'CONFIG_PINCTRL_SX150X=y'
-      - 'CONFIG_ROOT_NFS=y'
-      - 'CONFIG_RTC_DRV_CROS_EC=m'
       - 'CONFIG_SERIAL_8250_DW=y'
-      - 'CONFIG_SERIAL_8250=y'
-      - 'CONFIG_SND_DESIGNWARE_I2S=m'
-      - 'CONFIG_SND_DESIGNWARE_PCM=y'
-      - 'CONFIG_SND_HDA_CODEC_HDMI=m'
-      - 'CONFIG_SND_HDA_GENERIC=m'
-      - 'CONFIG_SND_SIMPLE_CARD=m'
-      - 'CONFIG_DRM_AMDGPU=y'
-      - 'CONFIG_DRM_AMD_ACP=y'
-      - 'CONFIG_SND_SOC_AMD_ACP=m'
-      - 'CONFIG_SND_SOC_AMD_CZ_DA7219MX98357_MACH=m'
-      - 'CONFIG_SND_SOC_CROS_EC_CODEC=m'
-      - 'CONFIG_SND_SOC_INTEL_BDW_RT5677_MACH=m'
-      - 'CONFIG_SND_SOC_INTEL_HASWELL=m'
-      - 'CONFIG_SND_SOC_INTEL_HASWELL_MACH=m'
-      - 'CONFIG_SND_SOC=m'
-      - 'CONFIG_SPI_PXA2XX=y'
-      - 'CONFIG_SPI=y'
-      - 'CONFIG_TCG_TIS_I2C_CR50=y'
-      - 'CONFIG_TCG_TIS_SPI_CR50=y'
-      - 'CONFIG_TCG_TIS_SPI=y'
-      - 'CONFIG_TCG_TIS=y'
-      - 'CONFIG_TCG_TPM=y'
-      - 'CONFIG_TOUCHSCREEN_ATMEL_MXT=m'
-      - 'CONFIG_USB_ETH=y'
-      - 'CONFIG_USB_GADGET=y'
-      - 'CONFIG_USB_RTL8152=y'
-      - 'CONFIG_USB_VIDEO_CLASS=m'
-      - 'CONFIG_WILCO_EC_DEBUGFS=m'
-      - 'CONFIG_WILCO_EC_EVENTS=m'
-      - 'CONFIG_WILCO_EC=m'
-      - 'CONFIG_WILCO_EC_TELEMETRY=m'
-      - 'CONFIG_X86_AMD_PLATFORM_DEVICE=y'
-      - 'CONFIG_X86_INTEL_LPSS=y'
-      - 'CONFIG_EXTRA_FIRMWARE="
-      amdgpu/green_sardine_asd.bin
-      amdgpu/green_sardine_ce.bin
-      amdgpu/green_sardine_sdma.bin
-      amdgpu/green_sardine_dmcub.bin
-      amdgpu/green_sardine_pfp.bin
-      amdgpu/green_sardine_vcn.bin
-      amdgpu/green_sardine_ta.bin
-      amdgpu/green_sardine_dmcub.bin
-      amdgpu/green_sardine_me.bin
-      amdgpu/green_sardine_mec.bin
-      amdgpu/green_sardine_mec2.bin
-      amdgpu/green_sardine_rlc.bin
-      amdgpu/raven2_asd.bin
-      amdgpu/raven2_ce.bin
-      amdgpu/raven2_gpu_info.bin
-      amdgpu/raven2_me.bin
-      amdgpu/raven2_mec2.bin
-      amdgpu/raven2_mec.bin
-      amdgpu/raven2_pfp.bin
-      amdgpu/raven2_rlc.bin
-      amdgpu/raven2_sdma.bin
-      amdgpu/raven2_ta.bin
-      amdgpu/raven2_vcn.bin
-      amdgpu/raven_kicker_rlc.bin
-      amdgpu/stoney_ce.bin
-      amdgpu/stoney_me.bin
-      amdgpu/stoney_mec.bin
-      amdgpu/stoney_pfp.bin
-      amdgpu/stoney_rlc.bin
-      amdgpu/stoney_sdma.bin
-      amdgpu/stoney_uvd.bin
-      amdgpu/stoney_vce.bin
-      i915/glk_dmc_ver1_04.bin
-      i915/kbl_dmc_ver1_04.bin
-      rtl_nic/rtl8153a-4.fw
-      rtl_nic/rtl8153b-2.fw
-      "'
 
```

Comment:
This change will break bootability of all chromebooks on mainline tests.
For example GPU tests (non-tast) need firmware files, etc.

Reply:
Maybe better to do this commit as follow-up, with separate testing on staging, making sure if removed options affecting or not chromebooks test results.

Reply:
ACK; new patch set only extends the fragment to support generic x86 boards as well